src/FsCodec.NewtonsoftJson/TypeSafeEnumConverter.fs

```diff
@@ -7,26 +7,28 @@ open System
 /// Utilities for working with DUs where none of the cases have a value
 module TypeSafeEnum =
 
-    let isTypeSafeEnum (t: Type) =
+    let isTypeSafeEnum (t : Type) =
         Union.isUnion t
         && (Union.getUnion t).cases |> Seq.forall (fun case -> case.GetFields().Length = 0)
 
-    let tryParseT (t: Type) (str: string) =
+    let tryParseT (t : Type) (str : string) =
         let union = Union.getUnion t
         union.cases
         |> Array.tryFindIndex (fun case -> case.Name = str)
         |> Option.map (fun tag -> (union.caseConstructor.[tag] [||]))
-    let tryParse<'T> (str: string) = tryParseT typeof<'T> str |> Option.map (fun e -> e :?> 'T)
 
-    let parseT (t: Type) (str: string) =
+    let tryParse<'T> (str : string) = tryParseT typeof<'T> str |> Option.map (fun e -> e :?> 'T)
```

Comment:
its fine, but I was trying to keep the two 'overloads' of parse and tryParse were being aligned visually the way it was?